File: src/utils/units.ts
// 단위 변환 유틸리티 (docs/TEMPLATE_SCHEMA.md 변환식 반영)

export const mmToPx = (mm: number, dpi: number): number => {
  return (mm / 25.4) * dpi;
};

export const pxToMm = (px: number, dpi: number): number => {
  return (px * 25.4) / dpi;
};

File: editor_core/units.py
from __future__ import annotations

MM_PER_INCH = 25.4


def mm_to_px(mm: float, dpi: float) -> float:
    """mm → px 변환. 반올림은 호출자가 결정한다."""
    return (mm / MM_PER_INCH) * dpi


def px_to_mm(px: float, dpi: float) -> float:
    """px → mm 변환."""
    return (px * MM_PER_INCH) / dpi

